snuba/settings_base.py

```diff
@@ -9,9 +9,45 @@
 PORT = 1218
 
 DEFAULT_DATASET_NAME = 'events'
-DISABLED_DATASETS = {}
 DATASET_MODE = 'local'
```

Comment:
Ideally we should eventually have structure in other parts of settings, but it would be way to big to do in one single PR